std/math.d

```diff
@@ -4809,12 +4905,13 @@ long lround(real x) @trusted nothrow @nogc
     version (Posix)
         return core.stdc.math.llroundl(x);
     else
-        assert(0, "lround not implemented");
+        static assert(0, "lround not implemented");
```

Comment:
Looks like the remainder isn't available on Win32, but still run?

```
std\math.d(4908): Error: static assert:  "lround not implemented"
```

Maybe we should roll a custom, "slow" implementation of it?